crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -328,6 +328,9 @@ pub unsafe trait WorldQuery {
     /// constructing [`Self::Fetch`](crate::query::WorldQuery::Fetch).
     type State: Send + Sync + Sized;
 
+    /// Runtime config value passed to [`WorldQuery::init_state`]
```

Comment:
This needs expansion for users. What is this type used for, and how should it be chosen?

Reply:
Done